spec/features/ips/add_ip_to_location_spec.rb

```diff
@@ -32,6 +32,22 @@
       end
     end
 
+    context 'with multiple IPs' do
+      before do
+        fill_in 'address', with: '10.0.0.1'
+        click_on 'Add another IP'
+        fill_in 'address2', with: '10.0.0.2'
```

Comment:
Had a bit of a play with capybara selectors, not sure how best to pick out the multiple select boxes when they come to be dynamically generated.